moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -284,12 +296,6 @@ const ProjectNotes = ({
       );
   };
 
-  /**
-   * Updates the note type based on conditions
-   * @param {Number} typeId
-   */
-  const changeFilterNoteType = (typeId) => setFilterNoteType(Number(typeId));
-
   // when the data changes, update the display notes state
   useEffect(() => {
```

Comment:
I didn't notice this when I was working on the validation but this `useEffect` and the one right below it really feel like they could be combined to replace the `displayNotes` state altogether if replace with a `useMemo` that handles the filtering. The `handleSnackbar` call could be removed too since there is an alert that renders on line 332 if notes fail to load.

Let me know what you think because you've already addressed a lot of tech debt in this one!

Reply:
Here's my pass at what I think you are suggesting, lemme know if this is what you were envisioning 
https://github.com/cityofaustin/atd-moped/pull/1577/commits/d31d48ad0cd1771edcd0a7713ad059d0fe7872af

Reply:
nice! one i think that one tiny change could eliminate the `displayNotes` state and `setDisplayNotes` setter. Thanks for updating btw! I think this will really help reduce confusion about where the data is coming from for the next person who dives in here.

```js
// return the notes inside of useFilterNotes instead of passing the setter
const displayNotes = useFilterNotes(mopedProjNotes, filterNoteType);
```

Reply:
thank you Mike! updated with this smarter way to do it

Reply:
thanks Chia!!! 🙏